.cirrus.yml

```
@@ -930,6 +930,21 @@ upgrade_test_task:
     always: *logs_artifacts
 
 
+fcos_image_build_task:
+    # Only a test. Does not push built images.
+    # Image push to quay will be handled by GHA in
+    # .github/workflows/fcos-podman-next-build.yml
+    name: "Test Build FCOS image $CTXDIR"
+    alias: fcos_image_build
+    only_if: $CIRRUS_PR != '' && $CIRRUS_BASE_BRANCH == 'main'
```

```suggestion
    only_if: $CIRRUS_PR != '' && $CIRRUS_BASE_BRANCH == 'main' && $CIRRUS_CHANGE_TITLE !=~ '.*CI:DOCS.*'
```

So all jobs except for `[CI:DOCS]`.  I think that's what you want?